manage.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
there's no need to add a `manage.py`

you can always use `DJANGO_SETTINGS_MODULE=tests.settings python -m django test` in your testing`